src/microbe_stage/organelle_components/LysosomeComponent.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Godot;
+
+public class LysosomeComponent : IOrganelleComponent
+{
+    public void OnAttachToCell(PlacedOrganelle organelle)
+    {
+        var configuration = organelle.Upgrades?.CustomUpgradeData;
+
+        var lipase = SimulationParameters.Instance.GetEnzyme("lipase");
+        var upgrades = configuration as LysosomeUpgrades;
+
+        var enzyme = configuration == null ? lipase : upgrades!.Enzyme;
```

Comment:
I think it would be slightly nicer to only look up the lipase enzyme if the configuration is null, just so we can save a tiny bit of performance without any drawback really here.